ONSdigital/ras-frontstage
# Csrf returning as 500

# Motivation and Context
When a csrf error usually occurs on RAS-Frontstage, a 500 error page would normally pop up with a 500 status code. This is the wrong way to handle it has the csrf expiry gives us a 400 response back. This PR adds a error handler to give a 400 status code back when it happens without a session. If it happens to a user who's logged in, it will then sign them out saying their session has expired.

This should hopefully improve analytics as well because the 500 status code was skewing the figures that we were getting.

# What has changed
- Added error handler for a CSRF error
# How to test?
- Run the tests
- In the testing config, you want to set `WTF_CSRF_ENABLED` to `True` and set `WTF_CSRF_TIME_LIMIT` to a second so it expires immediately. Outside of a session, an error page should appear when using a WTForm. When in a session, you should be logged out and given a message why.
# Links
[Trello](https://trello.com/c/k1M7FJAy)
